.github/workflows/release.yml

```diff
@@ -6,7 +6,26 @@ jobs:
     runs-on: windows-latest
     strategy:
       matrix:
-        mode: [osx-x64,linux-x64,win-x64]
+        mode: [osx-x64,linux-x64,win-x64,win-arm64,osx-arm64,linux-arm64]
```

Comment:
@Tatsinnit adding new arch should be another simple PR, issues raised by users were for arm64 so just kept arm64 for now. we can always support other arch's as well.

Reply:
Sounds good to me, I guess what I am eluding to here is that we should mentally be ready for arch miss in high is completely normal. Do we need pro-active gh issue open for it could be side thing to think of. 

☕️👍🛻

Reply:
@Tatsinnit sure we can add new issues for other arch's support like arm for raspberry Pi etc..